src/main/java/sboot/example/util/ReviewDtoMapper.java

```diff
@@ -0,0 +1,30 @@
+package sboot.example.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import sboot.example.dto.ReviewDto;
+
+@RequiredArgsConstructor
+public class ReviewDtoMapper {
+    private final CsvReaderUtil readerUtil;
+
+    public List<ReviewDto> getReviewDto() {
+        List<String[]> reviews = readerUtil.read();
```

Comment:
read what? let's pass a path(or file) to this method and use this util  class in CsvParserService along with mapper